### PR TITLE
Remove tests that depend on the CLI

### PR DESCRIPTION
**Pull request by littlejawa**
Remove integration tests that rely on `docker`, `docker-compose` or `swarn` on the CLI.

Fix issue #3270

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -166,56 +151,16 @@ cri-containerd:
...
 tracing:
 	bash tracing/tracing-test.sh
 
-time-drift:
-	bats integration/time_drift/time_drift.bats
-
-compatibility:
-	bash -f integration/compatibility/run.sh
-
-vcpus:
```
Comment (GabyCT) [inline Makefile:213]: the vcpus integration test is being updated to use ctr so it should not be remove it

Comment (littlejawa) [reply Makefile:213]: Got it - added it back.

**Review by GabyCT — APPROVED** [review]

**Comment by snir911** [comment]
Thanks Julian! LGTM

**Review by snir911 — APPROVED** [review]

**Pull request merged**